docs/src/making-changes.md

```diff
@@ -14,75 +14,88 @@ shell> git config --global user.email "EMAIL"
 
     If you forget to do this step until after you've already made some changes, don't worry: see [more detail about branching](@ref man-branch-post-hoc) below.
   * Make your changes. Whether it's fixing a bug or adding new functionality, in most cases your change should include updates to both the `src/` and `test/` folders. If you're fixing a bug, add your minimal example demonstrating the bug (on the current code) to the test suite; by contributing a test for the bug, you ensure that the bug won't accidentally reappear at some later time due to other changes. If you're adding new functionality, creating tests demonstrates to the package owner that you've made sure your code works as intended.
   * Run the package's tests and make sure they pass. There are several ways to run the tests:
 
-      * From Julia, run [`Pkg.test("Foo")`](@ref): this will run your tests in a separate (new) `julia` process.
-      * From Julia, `include("runtests.jl")` from the package's `test/` folder (it's possible the file has a different name, look for one that runs all the tests): this allows you to run the tests repeatedly in the same session without reloading all the package code; for packages that take a while to load, this can be much faster. With this approach, you do have to do some extra work to make [changes in the package code](@ref man-workflow-tips).
-      * From the shell, run `julia ../test/runtests.jl` from within the package's `src/` folder.
-
+    + From Pkg-REPL, run [`test Example`](@ref): this will run your tests in a separate (new) `julia` process.
+    + From Julia, `include("runtests.jl")` from the package's `test/` folder (it's possible the file has a different name, look for one that runs all the tests): this allows you to run the tests repeatedly in the same session without reloading all the package code; for packages that take a while to load, this can be much faster. With this approach, you do have to do some extra work to make [changes in the package code](@ref man-workflow-tips).
+    + From the shell, run `julia ../test/runtests.jl` from within the package's `src/` folder.
+  * If you are making substantial changes to the documentation, and the package documentation makes use of a dependency program to generate the documentation (e.g. Documenter.jl), you should test that your changes generate the correct documentation. For example, for `Documenter,jl`, from within the directory `Example.jl/docs`, you would invoke in shell mode `julia -e 'push!(LOAD_PATH,"../src/")' make.jl`
```

Comment:
`julia -e 'push!(LOAD_PATH,"../src/")' make.jl` this command clearly does not work but I have no idea how to say that one has to push to LOAD_PATH otherwise.